src/postgres/src/backend/parser/analyze.c

```diff
@@ -156,12 +159,6 @@ parse_analyze_varparams(RawStmt *parseTree, const char *sourceText,
 
 	query = transformTopLevelStmt(pstate, parseTree);
 
-	if (IsYugaByteEnabled())
```

Comment:
Why was this code deleted? If you have moved it somewhere else, can you also set queryid in `MyProc` ?